libraries/engage/back-in-stock/components/BackInStockButton/style.js

```diff
@@ -37,4 +36,9 @@ export default {
     alignSelf: 'flex-start',
     display: 'inline-flex',
   }).toString(),
+  iconCentered: css({
+    margin: 'auto',
```

Comment:
this causes that the whole element is right aligned when there is enough space

![Bildschirmfoto 2024-04-04 um 16 28 28](https://github.com/shopgate/pwa/assets/13087792/361b13d9-4d30-46b6-ae57-3f404637c413)